scipy/interpolate/_rbfinterp.py

```diff
@@ -387,6 +387,38 @@ def __init__(self, y, d,
         self.epsilon = epsilon
         self.powers = powers
 
+    def __chunk_evaluator(self,
+                          x,
+                          y,
+                          shift,
+                          scale,
+                          coeffs,
+                          memory_budget=None):
```

Comment:
Any reason for mangling the name? Just marking it private should be enough.

Also please describe the memory option.

For the indentation. The following is preferred as it minimizes diffs. Same applies bellow in functions/methods usages.
```suggestion
    def _chunk_evaluator(
        self,
        x,
        y,
        shift,
        scale,
        coeffs,
        memory_budget=None
    ):
```

Reply:
done